pep-0203.txt

```diff
@@ -5,342 +5,331 @@ Last-Modified: $Date$
-        PyNumber_InPlacePower(PyObject *o1, PyObject *o2);
-        PyNumber_InPlaceLshift(PyObject *o1, PyObject *o2);
-        PyNumber_InPlaceRshift(PyObject *o1, PyObject *o2);
-        PyNumber_InPlaceAnd(PyObject *o1, PyObject *o2);
-        PyNumber_InPlaceXor(PyObject *o1, PyObject *o2);
-        PyNumber_InPlaceOr(PyObject *o1, PyObject *o2);
-        PySequence_InPlaceConcat(PyObject *o1, PyObject *o2);
-        PySequence_InPlaceRepeat(PyObject *o, int count);
-
-    They call either the Python class hooks (if either of the objects
-    is a Python class instance) or the C type's number or sequence
-    methods.
-
-    The new bytecodes are:
-        INPLACE_ADD
-        INPLACE_SUBTRACT
-        INPLACE_MULTIPLY
-        INPLACE_DIVIDE
-        INPLACE_REMAINDER
-        INPLACE_POWER
-        INPLACE_LEFTSHIFT
-        INPLACE_RIGHTSHIFT
-        INPLACE_AND
-        INPLACE_XOR
-        INPLACE_OR
-        ROT_FOUR
-        DUP_TOPX
-
-    The INPLACE_* bytecodes mirror the BINARY_* bytecodes, except that
-    they are implemented as calls to the `InPlace' API functions. The
-    other two bytecodes are `utility' bytecodes: ROT_FOUR behaves like
-    ROT_THREE except that the four topmost stack items are rotated.
-
-    DUP_TOPX is a bytecode that takes a single argument, which should
-    be an integer between 1 and 5 (inclusive) which is the number of
-    items to duplicate in one block. Given a stack like this (where
-    the right side of the list is the `top' of the stack):
-
-        [1, 2, 3, 4, 5]
-
-    "DUP_TOPX 3" would duplicate the top 3 items, resulting in this
-    stack:
-
-        [1, 2, 3, 4, 5, 3, 4, 5]
-
-    DUP_TOPX with an argument of 1 is the same as DUP_TOP. The limit
-    of 5 is purely an implementation limit. The implementation of
-    augmented assignment requires only DUP_TOPX with an argument of 2
-    and 3, and could do without this new opcode at the cost of a fair
-    number of DUP_TOP and ROT_*.
+==============
+
+The current implementation of augmented assignment [2]_ adds, in addition to
+the methods and slots already covered, 13 new bytecodes and 13 new API
+functions.
+
+The API functions are simply in-place versions of the current binary-operation
+API functions::
+
+    PyNumber_InPlaceAdd(PyObject *o1, PyObject *o2);
+    PyNumber_InPlaceSubtract(PyObject *o1, PyObject *o2);
+    PyNumber_InPlaceMultiply(PyObject *o1, PyObject *o2);
+    PyNumber_InPlaceDivide(PyObject *o1, PyObject *o2);
+    PyNumber_InPlaceRemainder(PyObject *o1, PyObject *o2);
+    PyNumber_InPlacePower(PyObject *o1, PyObject *o2);
+    PyNumber_InPlaceLshift(PyObject *o1, PyObject *o2);
+    PyNumber_InPlaceRshift(PyObject *o1, PyObject *o2);
+    PyNumber_InPlaceAnd(PyObject *o1, PyObject *o2);
+    PyNumber_InPlaceXor(PyObject *o1, PyObject *o2);
+    PyNumber_InPlaceOr(PyObject *o1, PyObject *o2);
+    PySequence_InPlaceConcat(PyObject *o1, PyObject *o2);
+    PySequence_InPlaceRepeat(PyObject \*o, int count);
```

Comment:
@DimitrisJim , thank you so much for catching that!  The script converted them automatically and I hadn't checked for ones that needed to be changed back.  I've fixed this now and one in PEP238.